<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PVToolWidget</class>
 <widget class="QWidget" name="PVToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Weather">
     <property name="title">
      <string>Weather</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_WeatherComboBox">
        <property name="text">
         <string>Weather File</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_EPW">
        <property name="text">
         <string>EPW-File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_EPWFile"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_EPW">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_WeatherFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_Insulation">
     <property name="title">
      <string>Insulation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>367</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_RunSimu">
        <property name="text">
         <string>Run Simulation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_PVModule">
     <property name="title">
      <string>PV-Module</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1,0,1">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_PVDatabase">
        <property name="text">
         <string>PV Datenbank</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_PVModuleManufacture">
        <property name="text">
         <string>PV Herstellerdaten</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_uMPP">
        <property name="text">
         <string>Nennspannung (MPP) in V</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_uMPP"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_tempCoeffU">
        <property name="text">
         <string>Temperaturkoeffizient Uoc in %/K</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineEdit_beta"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_iMPP_2">
        <property name="text">
         <string>Nennstromst√§rke (MPP) in A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_iMPP"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_tempCoeffI">
        <property name="text">
         <string>Temperaturkoeffizient Isc in %/K</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineEdit_alpha"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_uOC">
        <property name="text">
         <string>Leerlaufspannung in V</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_uOC"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_tempCoeffP">
        <property name="text">
         <string>Temperaturkoeffizient PMPP in %/K</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="lineEdit_gamma"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_iSC">
        <property name="text">
         <string>Kurzschlussstrom in A</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_iSC"/>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_nSer">
        <property name="text">
         <string>Zellanzahl</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="lineEdit_nSer"/>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="comboBox_PVModule"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_PCM">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>PCM</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_PCMMaterials"/>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
